### ISSUE TITLE
STF is not starting after ./zebrunner.sh restore

### ISSUE THREAD

**Issue by dhreben** [zebrunner/mcloud-ios#80]
Test scenario:

1../zebrunner.sh backup
2. /zebrunner.sh shutdown
3.  /zebrunner.sh restore

Actual result: STF is not starting after ./zebrunner.sh restore

```
idmclouds-Mac-mini:mcloud-ios idmcloud$ tail -f ./logs/agents.log
18:26:24 Sync WDA script started
18:26:24 Sync Appium script started
18:26:25 Sync STF script started
Starting iSTF ios-device: b09fa26acc4c3f777e9b8b49e3348b7243f862b5 device name : iPhone_8
Now using node v8.17.0 (npm v6.14.8)
STF_CLI: /usr/local/lib/node_modules/@devicefarmer/stf/lib/cli
18:26:41 Sync Devices script started
18:26:44 Sync WDA script started
```

```

idmclouds-Mac-mini:mcloud-ios idmcloud$ tail -n 1000 -f ./logs/stf_iPhone_8.log
 --proxy-appium-port   Port allocated to Appium proxy
                            [number] [default: null]
 --device-name      Device name         [string] [default: false]
 --device-type      Device type         [string] [default: false]
Unknown argument: host
/usr/local/lib/node_modules/@devicefarmer/stf/lib/cli ios-device
```

**Comment by vdelendik** [comment]
fixed in scope of #81 improvements. `down` which is used by restore disable/unload services and guarantee we don't start services  in intermediate state when settings not restored correctly yet.

**Comment by dhreben** [comment]
Fixed
